test/week_test.js

```diff
@@ -57,6 +57,44 @@ describe('Week', () => {
     assert(day.prop('day').isSame(dayClicked, 'day'))
   })
 
+  it('should call the provided onWeekSelect function', () => {
+    let firstDayReceived = null
+    let weekNumberReceived = null
+
+    function onWeekClick (day, newWeekNumber) {
+      firstDayReceived = day.format()
+      weekNumberReceived = newWeekNumber
+    }
+
+    const weekStart = moment('2015-12-20')
+    const realWeekNumber = parseInt(weekStart.format('w'), 10)
+    const week = mount(
```

Comment:
shouldn't need to do a full mount, shallow better isolates the system under test